src/bindings/em/patch.cc

```diff
@@ -83,7 +84,7 @@ void change_set_noop(Patch::Change &change, T const &) {}
 EMSCRIPTEN_BINDINGS(Patch) {
   emscripten::class_<Patch>("Patch")
     .constructor<>()
-    .constructor<emscripten::val>(WRAP_STATIC(&constructor), emscripten::allow_raw_pointers())
+    .constructor(WRAP_STATIC(&constructor), emscripten::allow_raw_pointers())
```

Comment:
This change is related to https://github.com/emscripten-core/emscripten/issues/11274.